scaler/protocol/capnp/message.capnp

```diff
@@ -6,9 +6,10 @@ using Status = import "status.capnp";
 struct Task {
     taskId @0 :Data;
     source @1 :Data;
-    metadata @2 :Data;
-    funcObjectId @3 :Data;
-    functionArgs @4 :List(Argument);
+    tags @2 :List(Text);
```

Comment:
Tags are using capnp's text strings. Maybe we would want to use UTF-8 encoded bytes instead.